env/conjur-k8s/insecure/app.yml

```diff
@@ -35,15 +35,6 @@ spec:
     spec:
       containers:
       - image: cyberark/demo-app
-        securityContext:
```

Comment:
Ah, because it's in maintenance mode 🤦  Sorry. I'll fix that.
It was causing the DB to fail to configure based on the environment variables. It may be possible to work around but given that this is a demo that's only used in an ephemeral environment it doesn't seem worthwhile.